### PR TITLE
pip compatibility

### LINKED ISSUE #1: Python 3.12 support
Please add a pre-built wheel for Python 3.12

### PR DESCRIPTION
**Pull request by jstucke**
- makes the project directly installable with pip

resolves #1

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,12 +11,14 @@ license = "MIT"
...
-name = "entropython"
+name = "libentropython"
+path = "rust/lib.rs"
 crate-type = ["cdylib"]
 
 [dependencies.pyo3]
 version = "0.17.1"
-features = ["extension-module"]
 
+[features]
```
Comment (vobst) [inline Cargo.toml]: This change gets removed again in a later commit. Would be cleaner to have the final version here, i.e., the one uning the `[dependencies]` section.

File: Cargo.toml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [package]
 name = "entropython"
-version = "1.1.0"
+version = "1.1.1"
```
Comment (vobst) [inline Cargo.toml]: Should this also bump the Python package version?

**Review by vobst — APPROVED** [review]
Looks good to me. Only some minor nits.

File: Cargo.toml (hunk 2 above)
Comment (jstucke) [reply Cargo.toml]: I already bumped it from 1.0.2 to 1.1.0 in this PR

Comment (jstucke) [reply Cargo.toml]: Or do you mean bump it again?

**Review by vobst — APPROVED** [review]
Looks good to me now.

**Pull request merged**